pycbc/neutron_stars/pg_isso_solver.py

```diff
@@ -27,58 +27,31 @@
 from scipy.optimize import root_scalar
 
 
-def ISCO_eq(r, chi):
-    r"""Polynomial that enables the calculation of the Kerr innermost
-    stable circular orbit (ISCO) radius via its roots,
+def ISCO_solution(chi, incl):
+    r"""Analytic solution of the innermost
+    stable circular orbit (ISCO) for the Kerr metric.
 
-    .. math:: Z(r) = [r (r-6)]^2 - \chi^2 [2r (3r+14) - 9 \chi^2]\,.
+    ..See eq. (2.21) of
+    `Bardeen, J. M., Press, W. H., Teukolsky, S. A. (1972)
```

Comment:
I think this backtick is causing a failure in the automated tests.

```Warning, treated as error:
[51084](https://github.com/gwastro/pycbc/actions/runs/3421735975/jobs/5698287634#step:6:51085)
/home/runner/work/pycbc/pycbc/.tox/py-docs/lib/python3.10/site-packages/pycbc/neutron_stars/pg_isso_solver.py:docstring of pycbc.neutron_stars.pg_isso_solver:1:Inline interpreted text or phrase reference start-string without end-string.```

Reply:
Fixed, I think that the problem was that it could not find the closing backtick